common/src/main/java/com/zegelin/prometheus/cassandra/collector/StorageServiceMBeanMetricFamilyCollector.java

```diff
@@ -79,14 +79,18 @@ private StorageServiceMBeanMetricFamilyCollector(final StorageServiceMBean stora
         final Stream.Builder<MetricFamily> metricFamilyStreamBuilder = Stream.builder();
 
         {
-            final Stream<NumericMetric> ownershipMetricStream = storageServiceMBean.getOwnership().entrySet().stream()
-                    .map(e -> new Object() {
-                        final InetAddress endpoint = e.getKey();
-                        final float ownership = e.getValue();
-                    })
-                    .map(e -> new NumericMetric(metadataFactory.endpointLabels(e.endpoint), e.ownership));
-
-            metricFamilyStreamBuilder.add(new GaugeMetricFamily("cassandra_token_ownership_ratio", null, ownershipMetricStream));
+            try {
+                final Stream<NumericMetric> ownershipMetricStream = storageServiceMBean.getOwnership().entrySet().stream()
+                        .map(e -> new Object() {
+                            final InetAddress endpoint = e.getKey();
+                            final float ownership = e.getValue();
+                        })
+                        .map(e -> new NumericMetric(metadataFactory.endpointLabels(e.endpoint), e.ownership));
+
+                metricFamilyStreamBuilder.add(new GaugeMetricFamily("cassandra_token_ownership_ratio", null, ownershipMetricStream));
+            } catch (Throwable ex) {
```

Comment:
Minor nit: code style requires `final`?
(I should look into tooling to enforce this)